project.clj

```diff
@@ -1,7 +1,7 @@
 (def schema-tools-version "0.9.0")
 (def schema-version "1.1.3")
 
-(defproject threatgrid/ctim "0.4.3"
+(defproject threatgrid/ctim "0.4.4-SNAPSHOT"
```

Comment:
Is this the correct PR to be updating the version to SNAPSHOT? I presume that the version update is just piggy-backing on this PR